docs/source/installation.rst

```diff
@@ -13,12 +13,12 @@ See here for a `full list of precompiled wheels available on PyPI <https://pypi.
 .. contents::
    :local:
 
-For frequent issues with installation, consult the `Release versions - troubleshooting`_ section.
+For frequent issues with installation, consult the `Troubleshooting`_ section.
 
 There are three different installation types:
 
 * Installing sktime releases
-* Installing the latest sktime development version
+* Installing the latest sktime version
```

Comment:
I would add clear signpost that this is unstable. How about distinguishing:

first line - "stable releases"
second line - "unstable development version"

Reply:
Got it!